tests/loadtesting/README.md

```diff
@@ -0,0 +1,52 @@
+### Load testing
+
+##### Introduction
+we have applied `Load test` to test the performance of our admin endpoints, we have used `Locust` to  assess the performance.
+
+#### Technology Used
+we have used `Locust`.
+Locust is an easy-to-use, distributed, user load testing tool. It is intended for load-testing web sites (or other systems) and figuring out how many concurrent users a system can handle. [read more](https://docs.locust.io/en/stable/what-is-locust.html#features)
+
+##### Installation
+Locust is available on PyPI and can be installed with pip.
+``` bash
```

Comment:
please, mention it cannot be installed in poetry environment because we require a different `gevent` versions, hence, it should be installed outside of js-sdk environment.